PROJECT_PROGRESSION.md

```diff
@@ -225,7 +225,10 @@ To graduate from At Large stage or for a new project to join as an Impact projec
  * Adopt the Foundation Code of Conduct.
  * Accept security reports and publicly disclose them after the fix has been made available.
  * Other metrics as defined by the applying Project during the application process in cooperation with the CPC.
- * Receive a supermajority vote from the CPC to move to Impact stage. Projects can move directly from At Large to Impact, if they can demonstrate sufficient maturity and have met all requirements. 
+ * Obtain approval from the CPC to graduate to Impact stage, following the [decision-making][cpc charter decision making] and [voting][cpc charter voting] policies described in the CPC charter.
+    * An issue must be opened in the CPC repo which clearly states the project is being proposed for graduation to Impact stage.
+    * The issue must remain open for minimum of 14 days to allow for a sufficient feedback period.
```

Comment:
It feels like there's quite a bit of overlap with what's spelled out in [`GOVERNANCE.md`](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#merging-prs-into-this-repository). Could we point to relevant sections instead and maybe modify them to account for stage changes?

Reply:
My initial thought is that it's better to keep the acceptance criteria in one place as much as possible, but I'm open to moving things around if it makes sense.  Either way, my preference would be to land this change here (pending CPC approval), and follow up later/separately if we want to reorganize.

Reply:
Sorry missed your comment and offered the following changes to governance as a PR to your PR: https://github.com/openjs-foundation/cross-project-council/pull/1585

Reply:
Maybe it's better to remove the two sub-bullet points?  That would keep it limited to focusing on the specific change we explicitly wanted to make: changing the supermajority vote language to just following regular decision-making process.

Reply:
discussed in the CPC today and it was agreed removing the extra bullet points avoids muddying our governance further, and limits this PR just to the minimum change to update our process. we can follow up with broader changes in the future